### ISSUE TITLE
Does WarpDrive support...? also please implement....

### ISSUE THREAD

**Issue by GazbonicAcid** [LemADEC/WarpDrive#306]
Does WarpDrive work with Davinci's Vessels to allow me to actually "fly" my WarpDrive space ship?
I would also like to request you implement the "[MovingWorld](https://minecraft.curseforge.com/projects/movingworld/)" api to allow for native flying without going through Davinci's Vessels.

**Comment by LemADEC** [comment]
WarpDrive by itself is already compatible with it.
However, the library itself is corrupting ship during conversion, notably due to rotation.
Last time we've tried Davinci vessels on our server, we've also noted various issues that made inappropriate for public servers. Those issues goes deep into the fundation of the mod and, to my knowledge, would require a rewrite.
In short, use at your own risk.

We did a few prototype of the 'fly by' mode but performance were nowhere close to support at 20k blocks ships.
